tests/integration/security/egress_gateway_origination/testdata/destination-rule-tls-origination.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: originate-tls-for-edition-cnn-com
+spec:
+  host: edition.cnn.com
+  trafficPolicy:
+    loadBalancer:
+      simple: ROUND_ROBIN
+    portLevelSettings:
+      - port:
+          number: 443
+        tls:
+          mode: SIMPLE # initiates HTTPS for connections to edition.cnn.com
```

Comment:
I assume you will add a test for custom certificates here in the future? Two comments on this
* using custom certs with cnn.com probably won't work, so another reason not to use it
* We have an existing DR TLS test (direct, not using egress). It probably makes sense to unify these. See https://github.com/istio/istio/blob/c29027dd507cb2ef52fe590ab270d6a9a9f44b32/tests/integration/pilot/tls_test.go#L46

Reply:
yes those are specific to mTLS, which are TODO rn